shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -66,7 +66,10 @@
         {{> _loginButtonsSeparator}}
       {{/if}}
       <form class="email">
-        <label>Email <input name="email" type="email"> </label>
+        <label>Email
+         <label title="Passwordless E-mail Login allows you to log in using an e-mail address. No password is required; instead, every time you log in, we send you a new email confirmation code. You will only need to log in once per device." class="info">i</label>
```

Comment:
`<label>` has a specific meaning in HTML: it's supposed to be attached to an input element, either by having the input be a child of the label or by specifying it by ID. So I think you'll need to make this a `<span>`.